init-db/init-db.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
```

Comment:
I typically use `set -euf` to make scripts a little more bulletproof https://sipb.mit.edu/doc/safe-shell/